components/common-go/baseserver/server.go

```diff
@@ -151,13 +151,17 @@ func (s *Server) GRPCAddress() string {
 		return ""
 	}
 	addr := s.grpcListener.Addr().(*net.TCPAddr)
-	return fmt.Sprintf("%s:%d", addr.IP, addr.Port)
```

Comment:
`addr.IP` would render as `:::<port>` locally, which is not an address that gRPC likes